prefect_snowflake/database.py

```diff
@@ -566,7 +580,7 @@ async def snowflake_query(
     query: str,
     snowflake_connector: SnowflakeConnector,
     params: Union[Tuple[Any], Dict[str, Any]] = None,
-    cursor_type: SnowflakeCursor = SnowflakeCursor,
+    cursor_class: Optional[type[SnowflakeCursor]] = SnowflakeCursor,
```

Comment:
For the tasks, I think we should keep it as `cursor_type` to prevent breaking changes.

Reply:
Whoops, didn't read that super closely, i totally agree.